todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
```

Comment:
the content is not optional, you can not set the blank to True. Also, the content can be more than 15 characters, please set at least 60.